modules/core/include/opencv2/core/private.hpp

```diff
@@ -443,6 +442,121 @@ CV_EXPORTS void setUseTegra(bool flag);
 }
 #endif
 
+#ifdef ENABLE_INSTRUMENTATION
+namespace cv
+{
+namespace instr
+{
+struct InstrTLSStruct
+{
+    InstrTLSStruct()
+    {
+        pCurrentNode = NULL;
+    }
+    InstrNode* pCurrentNode;
+};
+
+class InstrStruct
+{
+public:
+    InstrStruct()
+    {
+        useInstr         = true;
+        enableMapping    = true;
+        pRefNode         = NULL;
+        threadRefCounter = 0;
+
+        rootNode.m_payload = NodeData("ROOT", TYPE_GENERAL, IMPL_PLAIN);
+        tlsStruct.get()->pCurrentNode = &rootNode;
+    }
+
+    Mutex mutex;
+
+    bool       useInstr;
+    bool       enableMapping;
+    InstrNode* pRefNode;
+    std::vector<InstrNode*> refVector;
+    int        threadRefCounter;
+    InstrNode  rootNode;
+    TLSData<InstrTLSStruct> tlsStruct;
+    std::vector<int> queue;
+};
+
+class CV_EXPORTS IntrumentationRegion
+{
+public:
+    IntrumentationRegion(const char* funName, const char* fileName, int lineNum, int instrType = TYPE_GENERAL, int implType = IMPL_PLAIN, bool mt = false);
+    ~IntrumentationRegion();
+
+private:
+    bool               m_disabled; // region status
+    unsigned long long m_regionTicks;
```

Comment:
There is defined "uint64" type in OpenCV (at least in cv namespace), so you can replace:
"unsigned long long" -> "uint64"